recipes/libpng/all/conanfile.py

```diff
@@ -37,6 +37,10 @@ class LibpngConan(ConanFile):
         "api_prefix": "",
     }
 
+    @property
+    def _is_clang_cl(self):
+        return str(self.settings.compiler) in ["clang"] and str(self.settings.os) in ['Windows']
```

Comment:
```suggestion
    @property
    def _is_clang_cl(self):
        return self.settings.os == "Windows" and self.settings.compiler == "clang" and \
               self.settings.compiler.get_safe("runtime")
```

Reply:
Would PRs that do the same on other problematic recipes be welcome or would you rather have users workaround this with local recipe forks? We have the same problem in other recipes and it's currently the only thing that prevents us from using the version from CCI

Reply:
I don't know. I think it would be better to convince conan team to really address issues like https://github.com/conan-io/conan/issues/13047 & https://github.com/conan-io/conan/issues/12336.